### PR TITLE
updating bleurt placeholder with the model

### PR DESCRIPTION
**Pull request by vaiibhavgupta**
Python files to inference using BLEURT have been added.

@jchoi92k will add the model because I was facing permission issues and was unable to push from jupyterhub or my local.

FYI, the optimal threhosld for this model is 0.6 as mentioned on line 8 of src/answer_eval.py file.

### CODE REVIEW

File: src/answer_eval.py [hunk 1]
```diff
@@ -1,11 +1,14 @@
 from models.answer import AnswerInput, AnswerResults
 from supabase import Client
+from bleurt import score as score_lib
```
Comment (langdonholmes) [inline src/answer_eval.py:3]: from bleurt.score import BleurtScorer

File: src/answer_eval.py [hunk 2]
```diff
@@ -23,19 +26,15 @@ def __init__(self, answer_input: AnswerInput, db: Client):
         self.answer = answer_input.answer
         self.results = {}
 
+        self.batch_size = 16
+        self.bleurt_scorer = score_lib.BleurtScorer(str(Path("assets/bleurt")))
```
Comment (langdonholmes) [inline src/answer_eval.py:30]: self.bleurt_scorer = BleurtScorer(str(Path("assets/bleurt")))

**Review by langdonholmes — CHANGES_REQUESTED** [review]
Vaibhav, thanks for your work on this. We have a ways to go before this can be merged.

- Use PyTorch instead of Tensorflow.
- It looks like you copy/pasted some code into assets. This code contains a bunch of checks and functionality that our system will never use, which increases the complexity of our codebase for no benefit. This won't work. You may either rewrite the code in a simpler form, using Pytorch, or alternatively, you can find a version of the code that runs on PyTorch and import that to your scoring script. You should also add this to requirements.txt.
- Please document and add any added requirements to requirements.txt
- Please confirm that the image will build before requesting review. You can test this by running the docker build command. As is, this pull request will not build.
- Large assets are handled by the download_models.py file. The BLEURT model should be uploaded somewhere (probably Huggingface), and you will need to modify download_models.py to fetch the model.

File: assets/bleurt/score.py
[the remark this answers is not in the document]
Comment (vaiibhavgupta) [reply assets/bleurt/score.py:1]: Like create a new folder "pipelines" in "textbook-summary-api" and put this "score.py" in there?

**Comment by langdonholmes** [comment]
I think this feature was added with a different PR

**Pull request closed without merging**